tests/integration/test_regression_multimol.py

```diff
@@ -0,0 +1,79 @@
+"""This integration test is designed to ensure that the chemprop model can _overfit_ the training
+data. A small enough dataset should be memorizable by even a moderately sized model, so this test
+should generally pass."""
+
+import warnings
+from pathlib import Path
+
+import pandas as pd
+import pytest
+import torch
+from lightning import pytorch as pl
+from torch.utils.data import DataLoader
+
+from chemprop import featurizers, nn
+from chemprop.data import (MoleculeDatapoint, MoleculeDataset,
+                           MulticomponentDataset, collate_multicomponent)
+from chemprop.models import multi
+
+N_COMPONENTS = 2
+pytestmark = pytest.mark.parametrize(
+    "mcmpnn", [(nn.BondMessagePassing(), N_COMPONENTS), (nn.AtomMessagePassing(), N_COMPONENTS)], indirect=True
+)
+
+@pytest.fixture
+def datas(mol_mol_regression_data):
+    smis1, smis2, Y = mol_mol_regression_data
+
+    return [[MoleculeDatapoint.from_smi(smi, y) for smi, y in zip(smis1, Y)], [MoleculeDatapoint.from_smi(smi) for smi in smis2]]
+
+
+@pytest.fixture
+def dataloader_scaler(datas):
+    dsets = [MoleculeDataset(data) for data in datas]
+    mcdset = MulticomponentDataset(dsets)
+    scaler = mcdset.normalize_targets()
+
+    return DataLoader(mcdset, 32, collate_fn=collate_multicomponent), scaler
+
+
+def test_quick(mcmpnn, dataloader_scaler):
+    dataloader, _ = dataloader_scaler
+    trainer = pl.Trainer(
+        logger=False,
+        enable_checkpointing=False,
+        enable_progress_bar=False,
+        enable_model_summary=False,
+        accelerator="cpu",
+        devices=1,
+        fast_dev_run=True,
+    )
+    trainer.fit(mcmpnn, dataloader)
+
+
+def test_overfit(mcmpnn, dataloader_scaler):
+    dataloader, scaler = dataloader_scaler
+    mcmpnn.loc = scaler.mean_
+    mcmpnn.scale = scaler.scale_
```

Comment:
Thanks for the comment! Yes, you're right, these should be the attributes of the `mcmpnn.predictor`. And you're right! Since the integration test scales the full dataset, setting this isn't necessary. Thanks for the catch! I will push a fix.